priv/www/js/tmpl/queue.ejs

```diff
@@ -295,14 +295,14 @@
       <input type="hidden" name="vhost" value="<%= fmt_string(queue.vhost) %>"/>
       <input type="hidden" name="name" value="<%= fmt_string(queue.name) %>"/>
       <input type="hidden" name="mode" value="delete"/>
-      <input type="submit" value="Delete" />
+      <input type="submit" value="Delete Queue" />
```

Comment:
I'm inclined to disagree - just because you and I know what the implication is, that doesn't mean new developers do. I don't see a problem with increasing clarity - it's the only time when the Delete expand shows two buttons. Unless someone's already had experience deleting an exchange, they might not catch on. It's more of a safety measure.